addons/web/static/src/views/list/list_controller.js

```diff
@@ -621,11 +621,7 @@ export class ListController extends Component {
         return true;
     }
 
-    onSavedMulti(records) {
-        records.forEach((record) => {
-            record.selected = false;
-        });
-    }
+    onSavedMulti(records) {}
```

Comment:
you can remove the function completely (and above in `get modelParams`) as the hook is no longer used.

Reply:
ok.